src/main/java/com/foundation/trello/model/List.java

```diff
@@ -0,0 +1,9 @@
+package com.foundation.trello.model;
+
+/**
+ * This class job with JsonConvert class.
```

Comment:
Please be clearer in the comment, you could use 'works' instead of 'job'.

Reply:
Fixed documentation